schemas/panels/stat/stat.cue

```diff
@@ -21,7 +21,7 @@ kind: "StatChart"
 spec: close({
 	query:       #ts_query
 	calculation: common.#calculation
-	unit:        common.#unit
```

Comment:
@sjcobb I think there is (was?) a misalignment between FE and BE here, is `unit` a mandatory or optional attribute on your end for StatChart? Same question applies to GaugeChart actually

Reply:
In FE it will default to 'Decimal' if it isn't set so it should be optional, I think I had it wrong before, my mistake!

Reply:
no worries! thanks for the confirmation :)